dvc/remote/s3.py

```diff
@@ -3,6 +3,7 @@
 import logging
 import os
 import threading
+from boto3.s3.transfer import TransferConfig
```

Comment:
Shouldn't this import be inside `_copy()`?

Reply:
good catch @skshetry ! I think it's better to move inside indeed. Mind creating a PR for this?

Reply:
@skshetry , got you) https://github.com/iterative/dvc/pull/3195

Good catch, indeed!

Reply:
Totally missed that one! Thank you @skshetry !